vtm.js

```diff
@@ -0,0 +1,339 @@
+
+//import './vtm.css';
+function exists(ref){
+  return (typeof ref) && (ref !== null);
+}
+
+
+/*TODO 
+    confirm whether data[0].id is the same for
+      strenght:{
+        %DATA%
+      } 
+      AND
+      {
+        id: "strenght"
+        %DATA%
+      }*/
+const app = Vue.createApp({
+  data() {
+    return {
+      skillPriorities:{
+        0:{ 
+          id: "specialist",
+          display: "Specialist",
+          resource: [0,2,3,3,1,0]
+        }
+      },
+      clans :{
+        thinblood:{
+          display: "Thin-blood",
+          abilities: null,
+          blood: "0",
+        },
+        caitiff:{
+          display: "Caitiff",
+          abilities: null,
+          blood: ">0"
+        },
+        nosferatu:{
+          display: "Nosferatu",
+          abilities: ["obfuscate", "potency", "animalism"],
+          blood: ">0"
+        },
+      },
+      bio: {
+        name : null,
+        age : null,
+        clanSelect: null,
+        bloodPotency : 0,
+        skillPrioritySelect: null,
+      },
+      //TODO^^DATA ABOVE IS WIP^^
+      attributes: {
+          display: "Attributes",
+          parent: "attributes",
+          data: [
+            {
+                display: "Physical",                 
+                list: [
+                    { 
+                      id: "strength",
```

Comment:
nerozumiem vyznamu, preco je tu id: "strength" malym a na dalsom riadku je display: "Strength" velkym

Reply:
no dobre, ale ako sa na to chces potom v kode odkazovat? podla pozicie?
priklad : willpower sa rovna wits + resolve alebo nieco take, teraz by to bolo po cesky takze by si mal "síla vůle" a hladal by si "rozum" a "odhodlání" alebo co je to v prekladu... proste predpokladam ze v internej podobe sa k tomu chces nejako dostat a display je hodnota ktoru zobrazis

Reply:
jo este jeden taky detail co som si vsimol u skills ze su radene abecedne, takze mozno by bolo u prekladu cool kebyze je tez radeny, a potom tam ztracas tu poziciu aj ked interne by to asi vypadalo stejne, ale proste bude potreba na to nejak odkazovat a dufam ze nechces po mne aby som si pametal ze Strenght je na prvej pozici v kategorii Physical pod Attributmi, ked sa na nu budem potrebovat na nejaky stat odkazovat